### DISCUSSION TITLE
SKIOVOX QUESTION

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#912] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
For the 120 patch for Skivox I cant load unpacked on the new thing.

**Comment by funkymokey** [comment]
did you extract the zip file?

**Reply by CGRmags** [reply]
@funkymokey I did it just doesnt show up when I press load unpacked

**Reply by CGRmags** [reply]
@funkymokey Like the file manager doesn't show up

**Comment by brobcoolguy** [comment]
after going into skiovox mode did you open up a new unblocked window? because im pretty sure the first unblocked window it opens up is a bit glitchy and you need to open a new window first

**Comment by CGRmags** [comment, marked as answer]
I figured it out Thanks!

On Mon, Feb 5, 2024 at 5:30 PM brobcoolguy ***@***.***> wrote:

> after going into skiovox mode did you open up a new unblocked window?
> because im pretty sure the first unblocked window it opens up is a bit
> glitchy and you need to open a new window first
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/912#discussioncomment-8377006>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBHHBUDHK7FIVHFVLO3YSGBRZAVCNFSM6AAAAABC2TIWCWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNZXGAYDM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
